src/main/external-resources/renderers/Sony-Bluray-BDP-S3700.conf

```diff
@@ -0,0 +1,47 @@
+#----------------------------------------------------------------------------
+# Profile for Sony Blu-ray Disc Players.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+# 7 September, 2019
+# Erik H. Beck
+#
+# Renderer for Sony Bluray BDP-S3700 (circa 2018 USA model)
+# Created from Sony-Bluray2013 config file
+#
+#
+RendererName = Sony Bluray BDP-S3700
+RendererIcon = Sony-Bluray2013.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# modelNumber=BDP-S3700
+# X-AV-Physical-Unit-Info: pa="BDP-S3700";
+#
+# ============================================================================
+#
+
+UserAgentSearch= BDP-S3700
+LoadingPriority = 2
+
+DLNALocalizationRequired = true
+TranscodeVideo = MPEGTS-MPEG2-AC3
```

Comment:
Could you please change this line to

```
TranscodeVideo = MPEGTS-H264-AC3
```

and see if that works?

Also please remove line 33, `HalveBitrate`, since that's more of a workaround setting